Jenkinsfile.d/core/weekly

```diff
@@ -5,9 +5,9 @@ pipeline {
     disableConcurrentBuilds()
   }
 
-  // Every Tuesday at 10:30:00 AM Coordinated Universal Time;
+  // Thursday at 9:30:00 AM Coordinated Universal Time - special case for one build;
   triggers {
-    cron '30 10 * * 2'
+    cron '30 9 * * 4'
```

Comment:
I don't think doing this will actually do anything, the build has to be run once for the cron trigger to be loaded.

Just trigger it manually

Reply:
Thanks!  I hadn't considered that.  Agreed that we'll trigger it manually rather than change the cron job and run it twice.